app/src/util/form-schema/manual-input-hierarchy.json

```diff
@@ -1490,6 +1560,835 @@
+            "vehicle-type-ferries",
+            "vehicle-type-boats",
+            "vehicle-type-marine-vessels"
+          ],
+          "multiselect": true
+        },
+        {
+          "id": "waterborne-navigation-fuel-source",
+          "type": "text"
+        }
+      ]
+    }
+  },
+  "II.3.2": {
+    "methodologies": [
+      {
+        "id": "electricity-consumption-waterborne-navigation-methodology",
+        "activities": [
+          {
+            "id": "electricity-consumption-waterborne-navigation-activity",
+            "unique-by": [
+              "waterborne-navigation-vehicle-type",
+              "waterborne-navigation-electricity-source"
+            ],
+            "extra-fields": [
+              {
+                "id": "waterborne-navigation-vehicle-type",
+                "options": [
+                  "vehicle-type-all",
+                  "vehicle-type-ferries",
+                  "vehicle-type-boats",
+                  "vehicle-type-marine-vessels"
+                ]
+              },
+              {
+                "id": "waterborne-navigation-electricity-source",
+                "type": "text"
+              }
+            ]
+          }
+        ],
+        "suggestedActivities": [
+          {
+            "id": "electricity-consumption-waterborne-navigation-activity",
+            "waterborne-navigation-vehicle-type": "vehicle-type-all"
+          },
+          {
+            "id": "electricity-consumption-waterborne-navigation-activity",
+            "waterborne-navigation-vehicle-type": "vehicle-type-ferries"
+          },
+          {
+            "id": "electricity-consumption-waterborne-navigation-activity",
+            "waterborne-navigation-vehicle-type": "vehicle-type-boats"
+          }
+        ]
+      }
+    ],
+    "directMeasure": {
+      "id": "waterborne-navigation-direct-measure-energy-consumption-methodology",
+      "extra-fields": [
+        {
+          "id": "waterborne-navigation-vehicle-type",
+          "options": [
+            "vehicle-type-ferries",
+            "vehicle-type-boats",
+            "vehicle-type-marine-vessels"
+          ],
+          "multiselect": true
+        },
+        {
+          "id": "waterborne-navigation-electricity-source",
+          "type": "text"
+        }
+      ]
+    }
+  },
+  "II.4.1": {
+    "methodologies": [
+      {
+        "id": "fuel-sales-aviation-methodology",
+        "inputRequired": ["total-fuel-sold-in-gas-stations"],
+        "activities": [
+          {
+            "id": "fuel-sales-aviation-activity",
+            "unique-by": [
+              "aviation-vehicle-type",
+              "aviation-fuel-type",
+              "aviation-fuel-source"
+            ],
+            "activity-title": "activity-total-fuel-sales",
+            "minimum": 1,
+            "extra-fields": [
+              {
+                "id": "aviation-fuel-type",
+                "options": [
+                  "fuel-type-gasoline",
```

Comment:
the correct fuel list:
- Aviation Gasoline
- Diesel
- Jet Gasoline
- Jet Kerosene
- Liquefied Petroleum Gas
- Gasoline
- Natural Gas Liquids

the change was applied in figma as well.